Source/OpenDRIVE/Private/OpenDRIVEActor.cpp

```diff
@@ -15,14 +15,16 @@ AOpenDRIVEActor::AOpenDRIVEActor()
 {
 	// Set this actor to call Tick() every frame.  You can turn this off to improve performance if you don't need it.
 	PrimaryActorTick.bCanEverTick = false;
-
+	SetRootComponent(CreateDefaultSubobject<USceneComponent>(TEXT("DefaultRootComponent")));
 }
 
 // Called when the game starts or when spawned
 void AOpenDRIVEActor::BeginPlay()
 {
 	Super::BeginPlay();
-	LoadOpenDrive();
+	
+	CheckForMultipleOpenDRIVEActors() ? GEngine->AddOnScreenDebugMessage(0, 10.f, FColor::Red, TEXT("Multiple OpenDRIVEActor detected. Must be only OpenDRIVEActor per scene. \n The plugin can't work correctly."))
```

Comment:
Sometimes I feel I don't understand you at all :P

Most of the time, you tend to be very verbose with if conditions (always adding `== true` when it's not necessary), which I kind of like actually, it makes code clearer and easier to understand.

But then you create this ternary which is just so out-of-place and would truly benefit from the verbose treatment.